lms/static/js/learner_dashboard/views/course_entitlement_view.js

```diff
@@ -313,9 +314,13 @@ defineFn(['common/js/vendor/bootstrap']);
                      this.$('.final-confirmation-btn:first').focus();
                  },
 
-                 hideDialog: function(invokingElement) {
+                 hideDialog: function(el, returnFocus = false) {
```

Comment:
You shouldn't need ```=  false``` because if it's not passed in it won't be true.

Reply:
Will remove it, was initially thinking it made it more explicit that it was an optional field.